content/about/feature-stages/index.md

```diff
@@ -26,29 +26,18 @@ within the project, not to the project as a whole. Here is a high level descript
 
 Below is our list of existing features and their current phases. This information will be updated after every monthly release.
 
-### Traffic management
+### Traffic Management
 
 | Feature           | Phase
 |-------------------|-------------------
-| [Protocols: HTTP 1.1](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http_connection_management.html#http-protocols)  | Beta
-| [Protocols: HTTP 2.0](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http_connection_management.html#http-protocols)  | Alpha
-| [Protocols: gRPC](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/grpc)   | Alpha
-| [Protocols: MongoDB](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/mongo)      | Alpha
-| [Request Routing](/docs/tasks/traffic-management/request-routing/)      | Alpha
-| [Fault Injection](/docs/tasks/traffic-management/fault-injection/)      | Alpha
-| [Traffic Shifting](/docs/tasks/traffic-management/traffic-shifting/)      | Alpha
-| [Circuit Break](/docs/tasks/traffic-management/circuit-breaking/)      | Alpha
-| [Mirroring](/docs/tasks/traffic-management/mirroring/)      | Alpha
-| [Ingress Traffic](/docs/tasks/traffic-management/ingress/)      | Alpha
-| [Egress Traffic](/docs/tasks/traffic-management/egress/)      | Alpha
-| [Egress TCP Traffic](/blog/2018/egress-tcp/)      | Alpha
-| [Improved Routing Rules: Composite Service](/docs/reference/config/istio.networking.v1alpha3/) | Alpha
-| [Quota / Redis Rate Limiting (Adapter and Server)](/docs/tasks/policy-enforcement/rate-limiting/) | Alpha
-| [Memquota Implementation and Integration](/docs/tasks/telemetry/metrics-logs/) | Stable
-| [Ingress TLS](/docs/tasks/traffic-management/ingress/) | Alpha
-| Egress Policy and Telemetry | Alpha
+| Protocols: HTTP1.1/HTTP2/gRPC/WebSockets/MongoDB/TCP  | Beta
```

Comment:
Two added words here need to be added to the TLD `.spelling` file.  I'm not sure which ones the linter doesn't like.  TCP, WebSockets, HTTP2 are all possibilities.  Might have to experiment to find the minimum set.

Cheers
-steve

Reply:
We use Websockets, not WebSockets